### ISSUE TITLE
Mikrotik and gpon onu

### ISSUE THREAD

**Issue by shpr0t** [Anime4000/RTL960x#54]
Hello, help with router os mikrotik setup.
how to configure to get access 192.168.1.1 web.
Dfp 34g-2c2 gpon onu  
sprut0x81gmail.com     my mail
thank you

**Comment by Anime4000** [comment]
With default mikrotik redefine config, do this

1. Go to Bridge > Ports, change `sfp-sfpplus1` to `ether1`
![image](https://user-images.githubusercontent.com/1908715/180143551-21a66575-c2b6-4fd4-9b37-f1c9bf917559.png)
2. Go to Interface > Interface List, add `sfp-sfpplus1` as WAN group
![image](https://user-images.githubusercontent.com/1908715/180143725-dee26094-cac8-4ec6-82b3-0246d93cc818.png)
3. Go to IP > Addresses, add IP Address `192.168.1.100/24` for `sfp-sfpplus1`
![image](https://user-images.githubusercontent.com/1908715/180143910-3ae2b226-6d47-445e-9d15-5fb3a4b53f3b.png)

Now you can access `192.168.1.1` from `192.168.88.0/24` subnet via NAT magic

**Comment by skon77** [comment]
> how to configure to get access on web interface?

My configuration for ROS7 on RB5009

1. Check that SFP1 is not in Bridge
2. In "IP / Interface List" add SFP1 to WAN group
3. In "IP / Address List" set "Address" as 192.168.1.2, "Network" as 192.168.1.1 and "Interface" as SFP1
4. In "IP / NAT rule" add "Chain" as <srcnat>, "dst.address" as 192.168.1.1, "Out.interface" as SFP1 and in "Action" tab set "Action" as <src-nat> and "To Addresses" as 192.168.1.2

**Comment by shpr0t** [comment]
one more question:
obligatory presence of an optical cable (fiber) to the module SFP for connection to 192.168.1.1 ?

**Comment by shpr0t** [comment]
received access, without optics does not connect.
thank you all

**Comment by rajkosto** [comment]
yes, this is a mikrotik problem only, it will not pass packets or try to establish link if the SFP module is reporting "RX Loss", which the stick does without a fiber connected.
if you use a TP-Link media converter or something for the SFP stick, you can access it's webgui even without fiber in.

**Comment by charlez79** [comment]
That can be resolved connecting the fiber to another SFP. That way i can get the module up and running anyway.
